app/controllers/api/users_controller.rb

```diff
@@ -24,7 +24,8 @@ def create
     user.update_facebook_avatar
     user.update_movies_from_facebook
 
-    render :json => user.to_json(:include => :elsewheres)
+    render :json => user.as_json(:include => :elsewheres)
+                        .merge(:has_completed_tour => user.recommendations.exists?)
```

Comment:
I wanted to do that, but then again a one line method was always very weird to me.